### DISCUSSION TITLE
php8.3-grpc Logs warnings since upgrade

### DISCUSSION THREAD

**Discussion by AleksCee** [oerdnj/deb.sury.org#2185] [Q&A]
Hi,

since the Upgrade: php8.3-grpc:amd64 (1.64.1-1+ubuntu22.04.1+deb.sury.org+1, 1.65.0-1+ubuntu22.04.1+deb.sury.org+1) yesterday the nextCloud cron.php comes with this warning at every run, was there a change in the loglevel or anything else? 

WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1721017985.296968 2890006 config.cc:230] gRPC experiments enabled: call_status_override_on_cancellation, event_engine_dns, event_engine_listener, http2_stats_fix, monitoring_experiment, pick_first_new, trace_record_callops, work_serializer_clears_time_cache, work_serializer_dispatch

thanks, Alex

**Comment by oerdnj** [comment]
You need to report this to gRPC project. The PHP gRPC extension is built with default options.

**Comment by AleksCee** [comment]
There seams to be already an issue: https://github.com/grpc/grpc/issues/37178 if I‘m right…?
